ci/launch_cluster.sh

```diff
@@ -10,74 +10,73 @@ if ! command -v envsubst >/dev/null 2>&1; then
     echo "envsubst was not found. Please install along with gettext."
     exit 1
 fi
+if ! command -v terraform >/dev/null 2>&1; then
+    echo "terraform was not found. Please install it."
+    exit 1
+fi
 
 # Two parameters are expected: CHANNEL and VARIANT where CHANNEL is the respective PR and
 # VARIANT could be one of three custer variants: open, strict or permissive.
-if [ "$#" -ne 3 ]; then
-    echo "Expected 3 parameters: launch_cluster.sh <channel> <variant> <deployment-name>"
+if [ "$#" -ne 4 ]; then
```

Comment:
maybe we should update the comment ☝️ 

Or remove it. Because the echo statement below is fine.